### DISCUSSION TITLE
Snap the bridge tool for paths to the water's surface

### DISCUSSION THREAD

**Discussion by HtotheTML** [OpenRCT2/OpenRCT2#21185] [Feature requests]
Similar to #17608, the construction mode for the path tool also snaps to underwater, rather than the water's surface. Would it be possible to have the start of a bridge begin on the water's surface, the same way rides can be built on water?

In addition, the construction grid for the paths should be drawn both on the water's surface as well as underwater, same as with the land and construction rights tools in #21150 for #18632.

**Comment by J-PIE-314** [comment]
PR #22569 added the ability to use ctr/shift modifiers and see the virtual grid during path placement. Does this satisfy your request?
